### PR TITLE
4958: HBX-Admin tool to enter APTC/CSR 

### PR DESCRIPTION
**Pull request by psk86**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [4958](https://devops.dchbx.org/redmine/issues/4958) |
| App-Dev Road Map # | 1.3.6.4 |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | n/a |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | @jackweiner |

### CODE REVIEW

File: app/helpers/insured/families_helper.rb [hunk 1]
```diff
@@ -97,9 +97,34 @@ def newhire_enrollment_eligible?(employee_role)
...
+        return true
+      end
+    end
+  end
+
+  def all_active_enrollment_with_aptc(family)
+    family.active_household.hbx_enrollments_with_aptc_by_year(TimeKeeper.datetime_of_record.year)
+  end
+
+  def hbx_memeber_names(hbx_enrollment_members)
```
Comment (Kevin-Wei) [inline app/helpers/insured/families_helper.rb]: @psk86 sp???

**Comment by davidplappert** [comment]
These four tickets need to be broken out. Each ticket is in a different state. This is causing your code to become stale. Thanks!

**Comment by Kevin-Wei** [comment]
Currently being tracked on https://github.com/dchbx/enroll/pull/689

**Comment by Kevin-Wei** [comment]
addressed via #689

**Comment by Kevin-Wei** [comment]
addressed via https://github.com/dchbx/enroll/pull/986

**Pull request closed without merging**